lib/phase/correction.cpp

```diff
@@ -15,10 +15,10 @@ Correction::Correction(ReachedNodes &reached_nodes)
   : Phase(reached_nodes),
     correction_participant(num_nodes())
 {
-  assert(std::any_of(reached_nodes.begin(),
+  /*assert(std::any_of(reached_nodes.begin(),
```

Comment:
Don't just comment out code. Either modify it appropriately, or remove it.

Reply:
Moreover, logically there are some nodes that in some sense are reached in the beginning of a correction phase. Aren't there?